Makefile

```diff
@@ -5,14 +5,15 @@ REGISTRIES ?= quay.io/cilium
 
 PUSH ?= false
 EXPORT ?= false
+PLATFORMs ?= linux/amd64,linux/arm64
```

Comment:
This should be `PLATFORMS` (with capital `S`). Looks like I merged the PR a bit to prematurely and the build on `master` branch currently [fails](https://github.com/cilium/image-tools/actions/runs/8363808916/job/22897656979) due to this. Fix at https://github.com/cilium/image-tools/pull/240

Reply:
Ah my mistake, thank you for catching this and taking care of the fix. I appreciate it.